src/bundle/Resources/views/content/tabs/locations.html.twig

```diff
@@ -38,7 +38,7 @@
                     {% endif %}
                 {% endif %}
             </td>
-            <td></td>
+            <td>{{ form_widget(actionsForm.locationVisibility[location.id]) }}</td>
```

Comment:
the field is completely useless in the `Location/Action` form. Since we only need a checkbox (and actually most likely more than that to have the widget slider to switch visibility) to trigger a submit of the `Location/Visibility` form, I  think we should remove the `locationVisibility` field from `Location/Action` and here directly generate a (disabled) checkbox input reflecting the visibility which we'll use to submit the other form. In addition, according to https://jira.ez.no/browse/EZP-27506, we should also display there if the Location is hidden because of one of its ascendant, so in the end we should have something like:

```twig
<input type="checkbox" name="visibility_{{ location.id }}" value="1" disabled
    {% if location.hidden %}checked{% endif %}>
{% if location.invisible %}
    <p>Hidden by superior</p>
{% endif %}
```